MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm

```diff
@@ -288,13 +288,17 @@
 										MVKRenderPass* renderPass,
 										MVKFramebuffer* framebuffer,
 										VkRect2D& renderArea,
-										MVKArrayRef<VkClearValue> clearValues) {
+										MVKArrayRef<VkClearValue> clearValues,
+										MVKArrayRef<MVKImageView*> imagelessAttachments) {
 	_renderPass = renderPass;
 	_framebuffer = framebuffer;
 	_renderArea = renderArea;
 	_isRenderingEntireAttachment = (mvkVkOffset2DsAreEqual(_renderArea.offset, {0,0}) &&
 									mvkVkExtent2DsAreEqual(_renderArea.extent, _framebuffer->getExtent2D()));
 	_clearValues.assign(clearValues.begin(), clearValues.end());
+	for(auto* v : imagelessAttachments) {
+		_imagelessAttachments.push_back(v);
+	}
```

Comment:
```suggestion
	_imagelessAttachments.assign(imagelessAttachments.begin(), imagelessAttachments.end());
```

Reply:
Hi, the compiler throws an error with new code:
```
MVKCommandBuffer.mm:299:31: Cannot initialize a parameter of type 'size_t' (aka 'unsigned long') with an rvalue of type 'MVKImageView *const *'
```
I think the compiler calls `assign(const size_t new_size, const Type *t)` rather than `assign(InputIterator first, InputIterator last)`.

Reply:
> I think the compiler calls `assign(const size_t new_size, const Type *t)` rather than `assign(InputIterator first, InputIterator last)`.

Oops...looks like the pointer version of `MVKSmallVectorImpl` doesn't have an iterator-based `assign()` function overload. Odd. 

If you feel up for doing some plumbing, you could add that function (based on the similar `assign()` overload in the non-pointer `MVKSmallVectorImpl` version). But otherwise I'm okay with leaving it as you have it.

Reply:
Sure, I'll add it.